presto-orc/src/main/java/com/facebook/presto/orc/OrcBatchRecordReader.java

```diff
@@ -41,8 +41,6 @@
 {
     private static final int INSTANCE_SIZE = ClassLayout.parseClass(OrcBatchRecordReader.class).instanceSize();
 
-    private final Map<Integer, Type> includedColumns;
```

Comment:
This variable was not used since the refactoring was done in @mbasmanova 's "Extract AbstractOrcRecordReader and StreamReader", where it was moved from OrcRecordReader to AbstractOrcRecordReader.